common/src/main/java/org/apache/sedona/common/raster/RasterAccessors.java

```diff
@@ -106,6 +106,12 @@ public static double getWorldCoordY(GridCoverage2D raster, int colX, int rowY) t
         return RasterUtils.getWorldCornerCoordinates(raster, colX, rowY).getY();
     }
 
+    public static double[] getWorldCoord(GridCoverage2D raster, int colX, int rowY) throws TransformException {
```

Comment:
The output should be a Geometry Point, not a double array.

Reply:
Done